migrations/programs-startDate-EndDate-6.0/setProgramsStartDateAndEndDate.js

```diff
@@ -0,0 +1,50 @@
+let _ = require("lodash");
+const path = require("path");
+let MongoClient = require("mongodb").MongoClient;
+let mongoUrl = process.env.MONGODB_URL;
+let dbName = mongoUrl.split("/").pop();
+let url = mongoUrl.split(dbName)[0];
+
+(async () => {
+    
+    let connection = await MongoClient.connect(url, { useNewUrlParser: true });
+    let db = connection.db(dbName);
+    try {
+        // get all active programs with createdAt value
+        let collectionDocs = await db.collection("programs").find({ 
+            status: "active",   
+            createdAt: {"$exists": true},
```

Comment:
@VISHNUDAS-tunerlabs Can we do this only for public program?

Reply:
@VISHNUDAS-tunerlabs if we don't have start and end date for pvt programs will that be an issue

Reply:
@aks30 front end requested for startDate for private programs also